### PR TITLE
Update InlineValidator::clientValidateAttribute()

### PR DESCRIPTION
**Pull request by WinterSilence**
Pass `$view` to callback `$this->clientValidate` to register view in user's asset

| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️/❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  |

### CODE REVIEW

**Comment by samdark** [comment]
That looks OK and actually what was planned long time ago. Would you please add a line for CHANGELOG? Thanks.

**Comment by WinterSilence** [comment]
@samdark done

File: framework/validators/InlineValidator.php [hunk 1]
```diff
@@ -100,7 +100,7 @@ public function clientValidateAttribute($model, $attribute, $view)
             if ($current === null) {
                 $current = $model->$attribute;
             }
-            return $method($attribute, $this->params, $this, $current);
+            return $method($attribute, $this->params, $this, $current, $view);
```
Comment (samdark) [inline framework/validators/InlineValidator.php:103]: phpdoc for `$clientValidate` property should be adjusted to mention `$view` in its signature. `$current` parameter seems to be missing as well in the example.

Comment (WinterSilence) [reply framework/validators/InlineValidator.php:103]: @samdark done

**Review by bizley — APPROVED** [review]

File: framework/validators/InlineValidator.php [hunk 2]
```diff
@@ -13,7 +13,7 @@
  * The validation method must have the following signature:
  *
  * ```php
- * function foo($attribute, $params, $validator)
+ * function foo($attribute, $params, $validator, )
```
Comment (machour) [inline framework/validators/InlineValidator.php:16]: 👀

**Review by Arhell — APPROVED** [review]

**Comment by bizley** [comment]
@WinterSilence are you preparing another approach for this?

**Comment by bizley** [comment]
Ok, cool.

@machour @samdark could you re-review it?

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Thanks.

**Pull request merged**